exercises/react/src/lib.rs

```diff
@@ -11,6 +11,12 @@ pub enum SetValueError {
     ComputeCell,
 }
 
+#[derive(Debug, PartialEq)]
+pub enum RemoveCallbackError {
+    NonexistentCell,
```

Comment:
note: It apparently is the case that we never test the `NonexistentCell` case. That can be dealt with later.